src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 17px;
+  border-radius: $border-radius-default;
+  border: 1px solid $color-white-smoke;
+
+  &__image {
+        width: 160px;
+        height: 134px;
+  }
```

Fix formating in all cases, just 2 space
```suggestion
  &__image {
    width: 160px;
    height: 134px;
  }
```